offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/GridUtils.groovy

```diff
@@ -79,7 +79,9 @@ class GridUtils {
             Date dateFromLocal = Date.from(localDate.atStartOfDay(ZoneId.systemDefault()).toInstant())
             return dateFromLocal == date
         }catch(Exception ignore){
-            return false
+            //if the local date cannot be parsed we want the filter to not be applied and return true for all inputs
+            return true
```

Comment:
While this solution works. I think it is misguiding to return true on mismatching dates. I will create a suggestion if this is fine with you?

Reply:
https://github.com/qbicsoftware/offer-manager-2-portlet/pull/511